api/replication.storage/v1alpha1/volumegroupreplicationcontent_types.go

```diff
@@ -49,10 +47,10 @@ type VolumeGroupReplicationContentSpec struct {
 
 	// VolumeGroupReplicationClassName is the name of the VolumeGroupReplicationClass from
 	// which this group replication was (or will be) created.
-	// +optional
+	// Required.
 	VolumeGroupReplicationClassName string `json:"volumeGroupReplicationClassName"`
 
-	// Source specifies whether the snapshot is (or should be) dynamically provisioned
+	// Source specifies whether the volume is (or should be) dynamically provisioned
```

Comment:
"volume group" ?

Reply:
No, it would be volume only because the source contains the list of volume handles part of volume group.

Reply:
i think it need to be as `volume group` as Niels mentioned above

Reply:
Yes we are using this source to create the volume group, that will we are specify the volume group is dynamically provisioned isn't it?  we might need to reword to convey that this is the source for the volume group not for volume or snapshot

Reply:
Ok, in that case does this sounds good?
```
// Source specifies whether the volume group is (or should be) dynamically provisioned
// or already exists using the volumes listed here, and just requires a
// Kubernetes object representation.
```